source/common/http/filter_manager.cc

```diff
@@ -875,13 +881,14 @@ void FilterManager::sendLocalReplyViaFilterChain(
               modify_headers(headers);
             }
           },
-          [this](ResponseHeaderMap& response_headers, Code& code, std::string& body,
-                 absl::string_view& content_type) -> void {
-            // TODO(snowp): This &get() business isn't nice, rework LocalReply and others to accept
-            // opt refs.
-            local_reply_.rewrite(filter_manager_callbacks_.requestHeaders().ptr(), response_headers,
-                                 stream_info_, code, body, content_type);
-          },
+          // Local reply rewrites (and upstream rewrites, for that matter) happen on the filter
+          // chain now, so we do not need to do any additional rewriting here.
+          /*rewrite_=*/nullptr,
+          // Local gRPC replies are encoded as trailers-only responses on the filter chain,
```

Comment:
For consistency with the above comment I should explicitly call out "but not upstream gRPC replies"

Reply:
Good call